### PR TITLE
Update record format

### PR DESCRIPTION
**Pull request by tsufz**
Revision of Record Format and Bump to version 2.4

### CODE REVIEW

File: Documentation/MassBankRecordFormat.md [hunk 1]
```diff
@@ -1165,6 +1173,20 @@ Example:
 AC$CHROMATOGRAPHY: TRANSFERLINE_TEMPERATURE 200 C
 ```
 
+#### <a name="2.4.7"></a>2.4.7 AC$GENERAL: subtag Description
+Experimental Methods and Conditions which are not included in other sections. Optional
+
+AC$GENERAL fields should be arranged by the alphabetical order of subtag names.
+
+##### 2.4.7 Subtag: CONCENTRATION
+Concentration of Analytical Standard Used for the Generation of MassBank Records. The mandatory concentration unit is mg/L.
```
[the remark this answers is not in the document]
Comment (tsufz) [reply Documentation/MassBankRecordFormat.md]: yes,

**Comment by tsufz** [comment]
@meier-rene and  @sneumann. Why is the new records not yet merged to the master? Do we need some implementations?

**Comment by meier-rene** [comment]
Hi @tsufz, I haven't merged this PR because it touches far more files than MassBankRecordFormat.md and I didn't took time to investigate whats going on here. Looks like a accidentally copy and paste of the whole Project source to the `compose` folder. To get this in fast, please answer the following two questions:
1. Should this PR update only the `MassBankRecordFormat.md`-file?
2. Are the commit messages important to you or can I just merge the content of `MassBankRecordFormat.md from my account?

**Comment by tsufz** [comment]
@meier-rene , it touches only the record format. I think there was line feed problem. The rest happened accidently....

**Comment by tsufz** [comment]
sorr for the unconviences...

**Comment by meier-rene** [comment]
Thank you for updating the record format. I have merged your changes to MassBankRecordFormat.md manually from your fork and committed with 6208e9a01560c28223cf981d45fd20216bd33fcb, thus closing your PR. Because your tsufz/MassBank-web:update_record_format branch has a broken codebase I suggest you delete it.

**Pull request closed without merging**